pkg/apis/elasticsearch/v1/elasticsearch_types.go

```diff
@@ -148,6 +148,10 @@ type TransportConfig struct {
 }
 
 type TransportTLSOptions struct {
+	// CommonNameSuffix when defined will be prefixed with the Pod name and used as the common name,
+	// and the first DNSName as well an OtherName required by Elasticsearch in the Subject Alternative Name extension of
+	// each Elasticsearch node's transport TLS certificate.
+	CommonNameSuffix string
```

Comment:
Naming: should this be called `OtherNameSuffix` because this is actually the relevant one (`CommonName` is only populated for legacy reasons)

Reply:
Yeah I think that makes sense to go with `otherName` 👍